support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -13,12 +13,6 @@ Parameters:
     Description: Stack name
     Type: String
     Default: support
-  OphanRole:
```

Comment:
What was this doing?

Reply:
This was needed when we used to call a lambda in the Ophan account to calculate annualised value for acquisitions. We haven't done this for ages but this had been missed. 

The main reason I've removed it in this PR was because otherwise I would have had to set a value for it from the AWS console to deploy this over the support-workers cdk deploy which was there where it had been removed - same for the Kinesis parameter which is also no longer used.